crates/ide/src/references/rename.rs

```diff
@@ -123,29 +122,23 @@ fn check_identifier(new_name: &str) -> RenameResult<IdentifierKind> {
     }
 }
 
-enum NameLike {
-    Name(ast::Name),
-    NameRef(ast::NameRef),
-    Lifetime(ast::Lifetime),
-}
-
```

Comment:
This was moved into `node_ext.rs` so `references.rs` ,`rename.rs` and `search.rs` can use it.